src/protocol/http.cc

```diff
@@ -1055,15 +1055,34 @@ std::string Request::get_header(const char *name) {
     char *pe = buffer_->str + header_length_;
 
     char *buffer = nullptr;
+    char *colon = nullptr;
 
     int state = 0;
+    int i = 0;
+
+    bool is_error_header_name = false;
+
     for (; p < pe; p++) {
```

Comment:
is memchr/strtok are better for updating p‘s offset?

Reply:
Thank yo for your review, I will try it later.